lib/perf_hooks.js

```diff
@@ -59,6 +59,15 @@ class Performance extends EventTarget {
       timeOrigin: this.timeOrigin,
     }, opts)}`;
   }
+
+}
+
+function toJSON() {
+  return {
+    nodeTiming: this.nodeTiming.toJSON(),
```

Comment:
I think this should just return `this.nodeTiming`, based on what Chrome does:
![image](https://user-images.githubusercontent.com/2352663/112116572-b4e4bf00-8bba-11eb-8bb8-56d2188ecfbe.png)

You see that `performance.toJSON()` returns instances of `PerformanceNavigation` and `PerformanceTiming` (which have their own `toJSON` methods)

Reply:
Ohh got it 👍🏽